data/summary/summary.go

```diff
@@ -21,12 +21,15 @@ const (
 
 type SummarizerRegistry struct {
 	summarizers map[string]any
+	Manager     *Manager
```

Comment:
I don't think the manager should be in the registry

Reply:
the manager is more accurately `TypelessSummarizer`, and ive renamed it as that, but GetSummarizer/addSummarizer are generics with specific requirements so I didnt see a clean way without another property.

ideally it would be added to the summarizers like everything else, but it doesnt have a GetSummary/UpdateSummary, since its all calculated live.

Reply:
continuous type was split from bgm/cgm, this is no longer relevant.